### PR TITLE
feat(theme): move aalborg theme into core

### PR DESCRIPTION
**Pull request by hypeJunction**
BREAKING CHANGE
Aalborg theme has been distributed among core views and plugins.
This may affect plugins that have relied on core styles,
or a certain priority of views
- [x] Update tests to not expect a theme as the last plugin

### CODE REVIEW

**Comment by hypeJunction** [comment]
Managed to salvage it. Decided to keep .php extensions until we know where we are with breaking down stylesheets.

File: engine/lib/navigation.php [hunk 1]
```diff
@@ -671,6 +671,10 @@ function _elgg_login_menu_setup($hook, $type, $return, $params) {
  */
 function _elgg_extras_menu_setup($hook, $type, $return, $params) {
 
+	if (!elgg_is_logged_in()) {
```
Comment (mrclay) [inline engine/lib/navigation.php]: Fine with this but I guess reportedcontent needs to check it.

Comment (hypeJunction) [inline engine/lib/navigation.php]: Aalborg was doing this, but can move to reported content.

Comment (jdalsem) [inline engine/lib/navigation.php]: lets keep it for now...

**Review by jdalsem — APPROVED** [review]

**Comment by hypeJunction** [comment]
I think should be good to go in. I've clicked around and all seems in order. Worst case we patch up later.

**Review by mrclay — APPROVED** [review]

File: engine/lib/views.php [hunk 2]
```diff
@@ -561,7 +561,7 @@ function _elgg_views_prepare_head($title) {
 	// https://developer.chrome.com/multidevice/android/installtohomescreen
 	$params['metas']['viewport'] = array(
 		'name' => 'viewport',
-		'content' => 'width=device-width',
+		'content' => 'width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0',
```
Comment (hypeJunction) [inline engine/lib/views.php]: That has always been in aalborg. Feel free to remove. I just want this out of the way so I can work on something more meaningful

**Comment by mrclay** [comment]
🎉

**Comment by jdalsem** [comment]
🎂 for everyone

**Pull request merged**